ArcGISRuntimeSDKQt_CppSamples/Analysis/FindServiceAreasForMultipleFacilities/FindServiceAreasForMultipleFacilities.cpp

```diff
@@ -176,3 +172,13 @@ void FindServiceAreasForMultipleFacilities::findServiceAreas()
     }
   });
 }
+
+void FindServiceAreasForMultipleFacilities::findServiceAreas()
+{
+  if (m_serviceAreaTask)
+    return;
+
+  m_serviceAreaTask = new ServiceAreaTask(serviceAreaTaskUrl, this);
+  connectServiceAreaTaskSignals();
```

Comment:
can this method only be called once?

Reply:
yes, the button is disabled after being pressed once